uploadNeuroDB/minc_deletion.pl

```diff
@@ -349,15 +358,19 @@ sub selORdel {
     my $nmi = $sth->fetchrow_array;
 
     if ($sth->rows > 0) {
-        my $new_nmi = $nmi - 1;
+        if ($field eq "SeriesUID") {
+            $nmi -= $sUIDFiles;
+        } else {
+            $nmi -= 1;
+        }
         $query = "UPDATE mri_upload SET number_of_mincInserted=? ".
             "WHERE TarchiveID=?";
 
         $sth = $dbh->prepare($query);
         my $success = $sth->execute($new_nmi, $tarchiveid);
```

Comment:
@davidblader 
you need to change the variable in the execution of the query as well :)